SPECS-EXTENDED/arpwatch/arpwatch.sysusers

```diff
@@ -0,0 +1,2 @@
+#Type Name      ID  GECOS                         Home directory     Shell
+u     arpwatch  -   "Service user for arpwatch"   /var/lib/arpwatch  /sbin/nologin
```

Comment:
Verified Introducing a new user.

Reply:
ahh .. sorry my notification on github are not showing up properly :(